_posts/2021-06-08-moveit-vs-moveit2.md

```diff
@@ -71,25 +86,25 @@ With the release of ROS 2 Galactic, we would like to share the new features only
     <tr>
       <td><a href="https://github.com/ros-planning/moveit2/pull/488" target="_blank">Hybrid Planning</a> (global + local planners)</td>
       <td class="not">✕</td>
-      <td class="done">✓</td>
+      <td class="done">pending</td>
     </tr>
     <tr>
-      <td>Realtime Support via DDS</td>
+      <td>Realtime Capable via DDS</td>
```

Comment:
```suggestion
      <td>DDS Messaging</td>
```

Reply:
@tylerjw this totally changes the meaning of this row. ROS 2 [supports realtime](https://docs.ros.org/en/foxy/Tutorials/Real-Time-Programming.html). DDS Messaging doesn't have much meaning to most audiences

Reply:
My point here was to be more honest about moveit.  We are using DDS but it would be disingenuous to say we are "realtime capable".

Reply:
Maybe "Based of ros2 which itself is realtime capable".

Reply:
Or maybe the status after "Realtime capable" should be "PRs welcome" instead of a checkmark.

Reply:
I'm trying to capture the idea that there is a possibility of Realtime because of DDS. I thought `Capable` was a little better wording as it doesn't say we support it, just that we could.